firmware/application/apps/ui_playlist.cpp

```diff
@@ -185,6 +193,10 @@ void PlaylistView::start() {
         button_play.set_bitmap(&bitmap_stop);
         baseband::set_sample_rate(sample_rate * 8);
 
+        if (now_delay != 0) {  // this `if` is because, if the delay is 0, it will sleep forever
```

Comment:
Not you could have use if( now_delay )

Reply:
Thank you! Sorry that if you don’t mind I’ll fix it next day🙏😆

Reply:
resolved